app/[pokemonName]/components/pokemon-details.jsx

```diff
@@ -29,12 +31,12 @@ export const PokemonPage = ({ pokemonData }) => {
   return (
     <div className={styles.container}>
       <Head>
-        <title>{`Pokémon Details - ${formatText(pokemonData.name)}`}</title>
+        <title>{`Pokémon Details - ${formatText(pokemonData?.name ?? 'Unknown')}`}</title>
```

Comment:
Is writing Unknown necessary here? On the server side, i guess you call notFound() when there isn't a pokemon with the name in the url